### PR TITLE
Gondolas are now basic mobs

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

This PR turns gondolas into basic mobs and does some fiddling with gondola pods
Their verbs are now action buttons which they lose upon delivering. If set to stay, they won't have their abilities afterwards (or wont get them in the first place if the delivery is already done).
Lets them survive without a pod requirement in case admins want to spawn one for their own stuff, so I also removed it from the snowflake checks to exclude them from stuff.

Also replaced the hardcoded "cant speak" with simply giving the mute trait, so admins can make gondolas speak if they want to for any reason.

## Why It's Good For The Game

The pet level of simple animals can finally be killed off, also gives admins more control over gondola-related stuff if they so wish.

## Changelog

:cl:
refactor: Gondolas (including gondola pods) are now basic mobs.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/basic/pets/gondolas/gondolapod.dm [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	gondola_owner.linked_pod.open_pod(gondola_owner, forced = TRUE)
+	for(var/datum/action/actions as anything in gondola_owner.actions)
+		if(actions.type in gondola_owner.gondola_delivering_actions)
+			actions.Remove(gondola_owner)
+	return TRUE
+
+///Checks the contents of the gondola and lets them know what they're holding.
+/datum/action/innate/check_gondola_contents
+	name = "Deliver"
+	desc = "Open your pod and release any contents stored within."
```
Comment (Jacquerel) [inline code/modules/mob/living/basic/pets/gondolas/gondolapod.dm:88]: You forgot to update these vars

**Review by Jacquerel — CHANGES_REQUESTED** [review]

**Comment by Higgin** [comment]
:DDD

**Review by san7890 — CHANGES_REQUESTED** [review]
very cool, thank you, just need to remove an HTML tag that doesn't work anymore

**Review by Maurukas — APPROVED** [review]
Mapdiff bot has been fixed so if you push a new commit of any type it will rerun.

This is just a replace paths though, so you don't really need to.

**Pull request merged**